person.rb

```diff
@@ -0,0 +1,19 @@
+class Person
+  attr_accessor :name, :age
+  attr_reader :id
+
+  def initialize(age, name = 'Unknown', parent_permission: true)
+    @id = Random.ran(1..1000)
```

Comment:
- [ ] You made a typo error here which results in the error below. According to the error message you used `ran`, instead of `rand`. Kindly fix this.

  ![tumwo-rand](https://user-images.githubusercontent.com/36542986/156177403-ba6e9fa9-4781-4776-b334-6eb3d6ac8c7f.jpg)